### ISSUE TITLE
[1.11.2] T2+ Blood Tanks revert to T1 capacity upon loading.

### ISSUE THREAD

**Issue by GeekBeam** [WayofTime/BloodMagic#1138]
#### Issue Description:
Blood Tanks 

#### What happens:

blood tanks of higher tiers occasionally lose their capacity, and revert back to capacity of the Tier one tank (16 buckets), voiding any fluid above that amount in the process

#### What you expected to happen:

that blood tanks of higher tiers would keep their capacity.

#### Steps to reproduce:

1. place Blood Tank of tier 2 or above.
2. unload the chunk containing the tank
3. check the capacity of the tank (recommended to use The One Probe for this.)
4. repeat until bug occurs.
...

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 2.1.9 - 78
- Minecraft: 1.11.2
- Forge: 13.20.1.2282

**Comment by TehNut** [comment]
Unable to reproduce.

https://streamable.com/efii1

**Comment by XZcutable** [comment]
Ye of little faith. He meant that the capacity, not the tier downgraded (I made the same mistake at first). I can assure you that this is in fact a bug and its worse than you think.

Here's how to reproduce:
https://streamable.com/btqqq

As for what happens, simply it seems that the tank is updating and taking the liquid, but in fact anything above 1600 mb (16 buckets) is visually taken by the client but when it updates (whether by opening a chest or using a bucket on the tank) it will update and show what is really in the tank and the buckets will still be full.

Edit:
So this only happens when they are place down and the chucks are unloaded (or world is reloaded). They work as expected at first until that happens.

**Comment by TehNut** [comment]
>Ye of little faith. He meant that the capacity, not the tier downgraded

If the steps to reproduce do not reproduce, then I cannot reproduce. I do not have the time to spend guessing what else is needed.

**Comment by XZcutable** [comment]
Ok here are the correct steps to reproduce.
1. Place down blood tank tier 2 or higher (only tested 2)
2. Unload the chunk (I reloaded the world).
3. Fill tank (I tested with lava)
4. Either reload the chunk again or use a bucket on tank. This will force it to actually update.

I refer to the video again:
https://streamable.com/btqqq

I also notice that when first placed down it will act like expected until the chunk is reloaded. Taking from the tank works but adding anything more than 16 does not work as expected (visually is does but not in actuality).
